exampleTop/configure/RELEASE

```diff
@@ -41,10 +41,7 @@ OPCUA = $(TOP)/..
 
 # Inside an embedded TOP, use the following definitions
 # pointing to the configuration of the parent module
--include $(TOP)/../../RELEASE.local
--include $(TOP)/../../configure/RELEASE.local
--include $(TOP)/../configure/RELEASE.local
--include $(TOP)/configure/RELEASE.local
+-include $(OPCUA)/configure/RELEASE
 
```

Comment:
This does not work as intended, because when including the parent's `RELEASE` file, `TOP` still points to the child module, so that the `.local` files relative to the parent are not included. (Which is the whole aim of the exercise.)

Replacing `$(TOP)/..` with `$(OPCUA)` is fine though, as the latter has been set to the first a few lines up.

Reply:
Using `$(OPCUA)` is actually not a good idea - see the note in line 31/33: if the parent module sets `INSTALL_LOCATION`, `OPCUA` points to that (which doesn't have a `configure` subdirectory) instead of `$(TOP)/..` (which has the configuration).

I think this needs to stay as it is.

Reply:
The whole thing is a convenience:

If you have an embedded Top in the original position (inside the OPCUA parent module), any `RELEASE.local` and `CONFIG_SITE.local` overrides of the parent should also apply to the embedded child module. (So that you configure e.g. `EPICS_BASE` only once in the parent and not in every child.)

If you copy-out the embedded Top and use it separately, it should act as a normal EPICS module and only use the `.local` overrides relative to its own position.